src/explore-education-statistics-admin/src/pages/methodology/__tests__/MethodologyPage.test.tsx

```diff
@@ -0,0 +1,184 @@
+} from '@admin/services/methodologyService';
+import _methodologyContentService, {
+  MethodologyContent,
+} from '@admin/services/methodologyContentService';
+import _permissionService from '@admin/services/permissionService';
+import { Router } from 'react-router';
+import { render, screen, waitFor, within } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { createMemoryHistory } from 'history';
+import React from 'react';
+
+jest.mock('@admin/services/methodologyService');
+jest.mock('@admin/services/methodologyContentService');
+jest.mock('@admin/services/permissionService');
+
+const methodologyService = _methodologyService as jest.Mocked<
+  typeof _methodologyService
+>;
+const methodologyContentService = _methodologyContentService as jest.Mocked<
+  typeof _methodologyContentService
+>;
+const permissionService = _permissionService as jest.Mocked<
+  typeof _permissionService
+>;
+
+describe('MethodologyPage', () => {
+  const testMethodology: BasicMethodology = {
+    id: 'm1',
+    amendment: false,
+    title: 'Test methodology',
+    slug: 'test-methodology',
+    owningPublication: {
+      id: 'p1',
+      title: 'Publication title',
+    },
+    status: 'Draft',
+  };
+  const testMethodologyAmendment = {
+    ...testMethodology,
+    amendment: true,
+  };
+
+  const testMethodologyContent: MethodologyContent = {
+    id: 'mc-1',
+    title: 'The content',
+    slug: 'content-1',
+    status: 'Draft',
+    content: [],
+    annexes: [],
+  };
+
+  test('renders the page with the summary tab', async () => {
+    methodologyService.getMethodology.mockResolvedValue(testMethodology);
+    renderPage();
+
+    await waitFor(() => {
+      expect(screen.getByTestId('page-title-caption')).toHaveTextContent(
+        'Edit methodology',
+      );
+      expect(screen.getByText('Test methodology')).toBeInTheDocument();
+
+      expect(
+        screen.getByRole('navigation', { name: 'tab-navigation' }),
+      ).toBeInTheDocument();
+
+      const primaryNav = screen.getByRole('navigation', {
+        name: 'tab-navigation',
+      });
+      const navLinks = within(primaryNav).getAllByRole('link');
+
+      expect(navLinks).toHaveLength(3);
+      expect(navLinks[0]).toHaveTextContent('Summary');
+      expect(navLinks[0].getAttribute('aria-current')).toBe('page');
+      expect(navLinks[1]).not.toHaveAttribute('aria-current');
+      expect(navLinks[2]).toHaveTextContent('Sign off');
+      expect(navLinks[2]).not.toHaveAttribute('aria-current');
+
+      expect(screen.getByText('Methodology summary')).toBeInTheDocument();
+    });
+  });
+
+  test('showing the status tag', async () => {
+    methodologyService.getMethodology.mockResolvedValue(testMethodology);
+    renderPage();
+
+    await waitFor(() => {
+      expect(screen.getByTestId('page-title-caption')).toHaveTextContent(
+        'Edit methodology',
+      );
+      expect(screen.getByText('Draft')).toBeInTheDocument();
```

Comment:
Could skip the `toBeInTheDocument` as we're asserting the class next (which must be on an existing element) anyway. Similar for the other test cases as well?